src/helpers/youtubeFetcher.js

```diff
@@ -1,30 +1,32 @@
 const fetch = require('node-fetch');
 const { YOUTUBE_API_KEY, YOUTUBE_CHANNEL_ID } = require('../config');
 
-function transformYoutubeResponseData({ items }) {
-  return items.map(({ snippet: { id: { videoID }, title, publishedAt, description, thumbnails: { high } } }) => {
-    return {
-      type: 'video',
-      name: title,
-      date: publishedAt,
-      url: `https://www.youtube.com/watch?v=${videoID}`,
-      thumbnail: high,
-      videoID,
-      title,
-      description,
-    };
-  });
-}
-
-export default async function fetchLatestYoutubeVideos() {
-  const url = `https://www.googleapis.com/youtube/v3/search?key=${YOUTUBE_API_KEY}&channelId=${YOUTUBE_CHANNEL_ID}&part=snippet&order=date&maxResults=50`;
+async function fetchLatestYoutubeVideos(maxResults) {
+  const apiUrl = `https://www.googleapis.com/youtube/v3/search?key=${YOUTUBE_API_KEY}&channelId=${YOUTUBE_CHANNEL_ID}&part=snippet&order=date&maxResults=${maxResults}`;
   try {
-    const resp = await fetch(url);
-    const data = await resp.json();
-    return transformYoutubeResponseData(data);
-  } catch (e) {
+    const resp = await fetch(apiUrl);
+    const { items } = await resp.json();
+    return items.map(({ id: { videoID }, snippet: { title, publishedAt, description, thumbnails: { high: { url } } } }) => {
```

Comment:
Can you change this structure from a block to a a one line assumed return? 
Can you alias `publishedAt` to `date`?
Can you alias `url` to `thumbnail`?

Reply:
Yes, the aliases are a good idea, is rlly confusing when you didn't write it yourself. Will update tonight (-3 GTM in my country)

Reply:
I agree!